metapool/sample_sheet.py

```diff
@@ -101,45 +128,66 @@ class KLSampleSheet(sample_sheet.SampleSheet):
         _ASSAY_KEY: None,
         'Description': '',
         'Chemistry': 'Default',
-    }
+    })
 
-    _READS = {
+    _READS = MappingProxyType({
         'Read1': 151,
         'Read2': 151
-    }
+    })
 
-    _SETTINGS = {
+    _SETTINGS = MappingProxyType({
         'ReverseComplement': '0',
         'MaskShortReads': '1',
         'OverrideCycles': 'Y151;I8N2;I8N2;Y151'
-    }
+    })
 
-    _ALL_METADATA = {**_HEADER, **_SETTINGS, **_READS,
-                     **_BIOINFORMATICS_AND_CONTACT}
+    _ALL_METADATA = MappingProxyType({
+        **_HEADER, **_SETTINGS, **_READS, **_BIOINFORMATICS_AND_CONTACT})
 
+    # TODO: Would love to know if this is being used outside this module or not
```

Comment:
I like to open all TODOs as issues so they do not get lost in the code; would that be possible? Another option could be to just make the change now. What do you think ?

Reply:
Sounds good, will do!  I'm deleting the TODO but leaving a brief comment in the code referencing the new issue; my reason for doing this is that, when I sit down to work on a chunk of existing code, I *don't* usually go scanning through the whole issue-base to see if any of them are suggestions relevant to that chunk that I could fix while I'm in there.  Given this, I worry about issues being where refactor suggestions go to die ;) .  Leaving a little breadcrumb to the issue will make it more likely that if I touch this code again, I remember to see if I implement its fix at that point.